Tests/Schema.NET.Test/StringExtensions.cs

```diff
@@ -1,13 +1,36 @@
-#if NET472 || NET461
 namespace Schema.NET.Test;
 
+#if NET472 || NET461
 using System;
-
-#pragma warning disable IDE0060 // Remove unused parameter
-#pragma warning disable CA1801 // Remove unused parameter
+#endif
+using System.IO;
+using Newtonsoft.Json;
```

Comment:
Not sure this is right - shouldn't this be `System.Text.Json`?

Reply:
I know this is just the test project but probably makes sense to use the same library throughout the project.

Reply:
Yes, I meant to do this but hadn't yet worked out how to implement the function using System.Text.Json. Leaving open.

Reply:
Ok figured it out.